src/include/common/task_system/progress_bar_display.h

```diff
@@ -14,11 +16,13 @@ class ProgressBarDisplay {
 
     virtual ~ProgressBarDisplay() = default;
 
-    // Update the progress of the pipeline and the number of finished pipelines
-    virtual void updateProgress(double newPipelineProgress, uint32_t newNumPipelinesFinished) = 0;
+    // Update the progress of the pipeline and the number of finished pipelines. queryID is used to
+    // identify the query
```

Comment:
Say queryID is needed when we track progress of multiple queries asynchronously.